test/image/mocks/axes_breaks-candlestick.json

```diff
@@ -0,0 +1,75 @@
+{
+    "data": [
+        {
+            "type": "candlestick",
+            "x": [
```

Comment:
in this mock, it would be great to see some data in the next visible range, like after 9am the next day, just to capture some of the tick labelling etc

Reply:
Good point. I added another mock in https://github.com/plotly/plotly.js/pull/4814/commits/8a4bae3333621ff4d75d1b82defe2613c959fbc8.
And we are going to revise tick behaviour (including these mocks) in #4734 after this PR is merged.